### DISCUSSION TITLE
Royalty hk.

### DISCUSSION THREAD

**Discussion by LadySmith** [Ludeon/RimWorld-Turkish#302] [General]
Proje uzun süredir (~2 ay) fazla ilerleme kat etmemiş.  Buna istinaden,

- [x] mevcut Royalty-Alpha-v0.2 dalını birleştirmek çakışmalar yüzünden zor. Ana dalı düzelterek denedim pek işe yaramadı, otomatik birleştiremiyoruz. Şimdilik elle kontrol ediyorum veya birleştireceğim.
- [x] bazı dosyalarda line ending problemleri mevcut, bu sebepten ötürü oyun dil dosyalarını görmeyebiliyor. Oyunun güncel dil dosyalarından düzeltmeye çalışıyorum. 

~~Bunları yaparken çok sık commit etmemeye özen göstereceğim. Olmazsa branch açarım.~~ 
Yeni branch veya PR açmak tekrar çakışmaya neden olacağı ve çalışma hızımı arttırmak için bunu es geçtim.

~~Lütfen dil dosyalarında ciddi değişikliğe gitmeden önce buraya yazın, yaptığınız çeviriler ve yaptığım düzeltmeler arada kaynamasın.~~
#300 başarılı bir şekilde birleştirilip güncellendi. Sıkıntı yok artık.

Şimdiye kadar çalışan herkesin ellerine sağlık. İyi ki varsınız arkadaşlar.

Edit1: Proje sayfası ekledim. Olabildiğince yaptıklarımızı/yapacaklarımızı takip etmek için. Tek yapmanız gereken eksik çevirileri yada fikirleri açarken gerekli tagleri kullanmak ve aynı yerden projeyi eklemek.

Edit2: 2 gün ve sıkı bir çalışmanın ardından merge ve update işleri tamamlandı. Artık eksik çeviri yok, çevrilmemiş çeviri var :)
Onlarıda zamanla yavaş yavaş hallederiz. Bunun üzerine [royalty-upgrade-project.md](https://github.com/Ludeon/RimWorld-Turkish/blob/71d59d6bebb48cfa37562a81de94e16596be8aed/royalty-upgrade-project.md)yi güvenle kaldırabilirim.

**Comment by nybadem** [comment]
(Bildirimlerdeki pull request numaraları karıştığı için buraya yazıyorum, umarım hata yapmamışımdır.)

Merhaba, şu sıralar büyük bir projede görevlendirildiğim için Nisan ayının sonuna kadar oyunun çevirisi ile ilgilenemeyeceğim. O yüzden mevcut durumda herhangi bir dosyayla uğraşamıyorum. Teslim zamanları çok sıkışık, oyunun çevirisine vakit ayıramıyorum :) Proje bittikten sonra çeviriye mümkün olduğunca hızlı bir şekilde geri döneceğim.

**Reply by LadySmith** [reply]
Cevabınız için teşekkürler. 
Elimden geldiğince repoyu royaltye göre ayarlayacağım. Proje sayfası ile ayrı bir dosya açıyorum şu sırada. Olurda geri döndüğünüzde çevrilmemiş bir yer kalırsa sizi görmeyi çok isterim. 
Görüşmek üzere. Kolay gelsin. Şimdiden elinize sağlık :)
